Add helm chart to be used by Gardener-Addon to provide Kyma on top of gardener cluster (#38)
kyma-incubator/marketplaces: gardener-addon/README.md

> # Kyma Gardener Addon

Reviewer: ```suggestion
# Kyma Add-On for Gardener
```
In Gardener, it's just `Kyma Add-On` so I would leave it like that.

## Overview

> This project provides a helm chart to be used as Addon for [Gardener](https://gardener.cloud)

Reviewer: ```suggestion
This project provides a Helm chart you can use as an Add-On for [Gardener](https://gardener.cloud).
```

> It will install a set of Kubernetes Jobs installing and starting the Kyma-Installer.

Reviewer: ```suggestion
The chart installs a set of Kubernetes Jobs installing and starting the Kyma Installer.
```

## Prerequisites

Use the following tools to set up the project:

* [Helm](https://helm.sh)

## Usage

> To install the addon manually on a Gardener cluster, follow these steps:

Reviewer: ```suggestion
To install the Kyma Add-On manually on a Gardener cluster, follow these steps:
```

> 1. Render helm chart:

Reviewer: ```suggestion
1. Render the Helm chart:
```

    ```bash
    helm template . > output.yaml
    ```

> 2. Apply the kubernetes resources:

Reviewer: ```suggestion
2. Apply Kubernetes resources:
```

    ```bash
    kubectl apply -f output.yaml
    ```

3. Check jobs for healthy execution:

    ```bash
    kubectl -n kyma-installer get jobs -w
    ```

4. Check installation progress:

    ```bash
    kubectl -n kyma-installer logs kyma-initializer-dns-XXX -c wait-for-installation -f
    ```

5. Check for final completion
    ```bash
    kubectl -n kyma-installer logs kyma-initializer-dns-XXX
    ```

## Details

> For doing the steps of the job in a manual way, see the [manual install](manual-install.md) instructions.

Reviewer: ```suggestion
If you want to execute the job manually, see the [manual installation](manual-install.md) instructions.
```
